corehq/apps/case_search/xpath_functions.py

```diff
@@ -35,6 +38,32 @@ def date(node):
     )
 
 
-XPATH_FUNCTIONS = {
+def _selected_query(node, fuzzy, operator):
+    if len(node.args) != 2:
+        raise XPathFunctionException(_(f"The {node.name} function accepts exactly two arguments."))
+    property_name = serialize(node.args[0])
+    search_values = node.args[1]
+    return case_property_query(property_name, search_values, fuzzy=fuzzy, mode=operator)
+
+
+def selected(node, fuzzy):
+    return _selected_query(node, fuzzy=fuzzy, operator='or')
+
+
+def selected_any(node, fuzzy):
```

Comment:
since this is identical to `selected` you could delete it and alias `selected` to `selected-any` in the function map